content/blog/2025-11-05-valkey-helm-chart/index.md

```diff
@@ -0,0 +1,141 @@
+
+To reduce the impact on Valkey deployments, the community created an official, project-owned Helm chart (**request: [issue #2371](https://github.com/valkey-io/valkey/issues/2371)**, **chart: [valkey-helm](https://github.com/valkey-io/valkey-helm)**). With the official chart, you can pin chart and image versions, keep `values.yaml` in code, and upgrade on your schedule without depending on vendor policy changes.
+
+### Why a Valkey maintained chart helps
+
+With the official chart, you run what you intend, not what a third party changes. Pin a chart release from the Valkey repo (for example `--version 0.7.7` from [https://github.com/valkey-io/valkey-helm](https://github.com/valkey-io/valkey-helm)) and lock the Valkey image tag in your `values.yaml`. Because the chart follows Valkey releases and docs, you can bump versions in a pull request, test in staging, then promote the same versions to production. If something breaks, use Helm history to roll back with `helm rollback <release> <revision>`. Keep your values in source control, often per environment (`values.staging.yaml`, `values.prod.yaml`), and you get a clean GitOps flow. For details and examples, see the README: https://github.com/valkey-io/valkey-helm#readme.
+
+
+### Essential capabilities in the Valkey Helm Chart.
+
+This chart focuses on the basics. It gives you a clean default and a few knobs you actually need.
+
+* **Small cache for one service.** One Valkey pod, no persistence, ClusterIP service. Fast to start. Good for stateless caches. See the install steps in the README: [https://github.com/valkey-io/valkey-helm](https://github.com/valkey-io/valkey-helm).
+* **Read-heavy traffic.** One primary with two replicas. Point reads at a separate service if you prefer. Writes still go to the primary. Configure replicas in `values.yaml` and upgrade with Helm.
+* **Simple durability.** Turn on PVCs. Pick a storage class your cluster supports. Back up PVCs with your platform’s snapshot tools while chart-level hooks are being designed.
+* **Safe deploys.** Pin your chart version and Valkey image tag in `values.yaml`. Use `helm diff` in CI. If probes fail after an upgrade, roll back immediately with Helm.
+
+## How to migrate from Bitnami to the Valkey Helm chart (in-place).
+
+**This path has downtime.** Plan a short **maintenance window** and pause writers during the swap.
```

Comment:
You use `acme-valkey` as a namespace through out this example. You should put a note that you'll need to swap it out with a real namespace.

(Readers tend to just copy/paste stuff and you either need to be explicit that you're using a made up name. Alternately, you make it really work by using an environment variable so it becomes very copy/paste friendly)